src/cascadia/TerminalApp/CommandPalette.xaml

```diff
@@ -43,6 +42,70 @@ the MIT License. See LICENSE in the project root for license information. -->
                                   AutomationProperties.Name="{x:Bind Item.Name, Mode=OneWay}"
                                   AutomationProperties.AcceleratorKey="{x:Bind Item.KeyChordText, Mode=OneWay}">
 
+                    <Grid HorizontalAlignment="Stretch" ColumnSpacing="8" >
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="16"/>
+                            <!-- icon -->
+                            <ColumnDefinition Width="Auto"/>
+                            <!-- command label -->
+                            <ColumnDefinition Width="*"/>
+                            <!-- key chord -->
+                            <ColumnDefinition Width="16"/>
+                            <!-- gutter for scrollbar -->
+                        </Grid.ColumnDefinitions>
+
+                        <IconSourceElement
+                                    Grid.Column="0"
+                                    Width="16"
+                                    Height="16"
+                                    IconSource="{x:Bind Item.Icon,
+                                                        Mode=OneWay,
+                                                        Converter={StaticResource IconSourceConverter}}"/>
+
+                        <local:HighlightedTextControl
+                                    Grid.Column="1"
+                                    HorizontalAlignment="Left"
+                                    Text="{x:Bind HighlightedName, Mode=OneWay}"/>
+
+                        <!-- The block for the key chord is only visible
+                                when there's actual text set as the label. See
+                                CommandKeyChordVisibilityConverter for details.
+                             We're setting the accessibility view on the
+                                border and text block to Raw because otherwise,
+                                Narrator will read out the key chord. Problem is,
+                                it already did that because it was the list item's
+                                "AcceleratorKey". It's redundant. -->
```

Comment:
I do not believe that you can set a key chord on a setting with nested settings under it -- @zadjii-msft, can you?

Reply:
Nevermind, this isn't even the same template. Question stands though!

Reply:
Maybe we _want_ nested commands to have key chords in the future? So that when you press them they just _open up a menu directly to the command palette filtered to the subcommands???_

Reply:
That's what I want, tbh. keybinding --> submenu.

Reply:
I think that was a "future consideration" for the command palette, but never _really_ thought through too hard